button_res.py

```diff
@@ -0,0 +1,20 @@
+import tkinter as tk
+
+
+class App(tk.Tk):
+    def __init__(self):
+        super().__init__()
+        self.geometry("700x600")
+        self.title("Minesweeper")
+        but1 = Button_play()
+        but1.pack()
+
+
+class Button_play(tk.Button):
```

Comment:
Назви класу прийнято називати в стилі CamelCase (великі літери замість `_`)
```suggestion
class PlayButton(tk.Button):
```